docs/restic_troubleshooting.md

```diff
@@ -0,0 +1,363 @@
+# velero restic troubleshooting tips
+
```

Comment:
Maybe add a sub heading of Velero restic FS copy

Reply:
```suggestion
## Velero restic FS copy
<!--TODO: -->
```